### PR TITLE
Add warning messages for missing directory and skipped files

### PR DESCRIPTION
**Pull request by ErnstHaagsman**
I just started up a project, and found this tool for managing migrations. It looked like the simple tool I was looking for to apply migrations. 

After installing it, creating a 'migrations' folder, and writing my first migration in `V0001_initial_migration.sql`. pgmigrate returned nothing when applying the migration, and I wasn't sure if it had succeeded or failed. I could see in the database that although the information table was added, none of my tables were.

I tried to figure out what went wrong, and added `--verbose`. And still I got no error message, but also no tables.

Finally, when stepping through the code I noticed that in the code all files that don't match the regex get silently ignored, even when adding `--verbose`.

I hope that with the proposed additional log and error messages it would be easier for newcomers to see what's happening when they get something wrong.

### CODE REVIEW

File: features/steps/migration_dir.py [hunk 1]
```diff
@@ -14,3 +14,12 @@ def step_impl(context):
     context.migr_dir = tempfile.mkdtemp()
     os.mkdir(os.path.join(context.migr_dir, 'migrations'))
     os.mkdir(os.path.join(context.migr_dir, 'callbacks'))
+
+
+@given('no migration dir')
```
Comment (secwall) [inline features/steps/migration_dir.py:19]: This does not look like no migratiions dir for me. More like empty one.

File: features/errors_handle.feature [hunk 2]
```diff
@@ -144,3 +144,8 @@ Feature: Handling migration errors
         Then pgmigrate command "failed"
         And migrate command failed with First migration MUST be transactional
         And database has no schema_version table
+        
+    Scenario: Missing Migrations directory
+        Given no migration dir
+        When we run pgmigrate with "-t 1 migrate"
+        Then migrate command failed with Could not find migrations directory
```
Comment (secwall) [inline features/errors_handle.feature]: We probably should add newline at the end of this file.

File: doc/tutorial.md [hunk 3]
```diff
@@ -14,6 +14,10 @@ foodb
 │   └── beforeEach # will be executed before each migration
 ├── grants # use this dir to set special callbacks for grants
 ├── migrations # migrations dir
+│   ├── V0001__Initial_schema_foo.sql    # Migrations should have unique,
```
Comment (secwall) [inline doc/tutorial.md:17]: We probably should drop this change from commit or change commit message.

File: pgmigrate.py [hunk 4]
```diff
@@ -184,6 +188,12 @@ def _get_migrations_info_from_dir(base_dir):
                     '\nfirst : %s' % migration.filePath +
                     '\nsecond: %s' % migrations[version].filePath)
             migrations[version] = migration
+    else:
+        LOG.error(
+            "Could not find 'migrations' directory. Did you specify the "
```
Comment (secwall) [inline pgmigrate.py]: This message is meaningless, we probably should just raise an exception here.

**Review by secwall — CHANGES_REQUESTED** [review]

File: doc/tutorial.md (hunk 3 above)
Comment (ErnstHaagsman) [reply doc/tutorial.md:17]: I've split the commit, separating this change into a commit with a more appropriate message

File: pgmigrate.py (hunk 4 above)
Comment (ErnstHaagsman) [reply pgmigrate.py]: I've removed it

File: features/steps/migration_dir.py (hunk 1 above)
Comment (ErnstHaagsman) [reply features/steps/migration_dir.py:19]: The temporary directory created here is the 'project root' which can have any name, and contains the `migrations.yml` file. pgmigrate.py expects this folder to have a `migrations` subfolder, with exactly that name.

**Comment by ErnstHaagsman** [comment]
I think the split commit may have confused GitHub. I've made the requested changes. The only thing I'm not sure about is the comment about the empty migrations dir.

**Comment by secwall** [comment]
Hm. It really seems that github was confused by splitting. I've commited changes based on this pull request as 76c828b and bab18b9.

**Pull request closed without merging**